views/confirmation-not-required.html

```diff
@@ -24,16 +28,17 @@ <h1 class="govuk-panel__title">You've told us
         <h2 class="govuk-heading-m">What happens next</h2>
         <p>You should
             <a href="http://resources.companieshouse.gov.uk/close-a-company/index.html">close the company</a>
-            so that it is removed from the register. You do not need to file the
-            {{ overdueFiling }}
-            before closing the company.</p>
+            so that it is removed from the register. When you have closed the company, we will no longer pursue the overdue {{ overdueFiling }}.
         </p>
 
-        <p>
-            {{company.companyName}}
-            will receive a
-            <a href="https://www.gov.uk/annual-accounts/penalties-for-late-filing">late filing penalty</a>
-            that increases over time until the company is closed.</p>
+        <p>If you do not close the company and do not file your overdue {{ overdueFiling }}, we may prosecute the company directors.</p>
+
+        {% if (overdueFiling === 'accounts') or (overdueFiling === 'accounts and confirmation statement') %}
```

Comment:
it would be better to pass in a boolean flag here to indicate if this text block needs to be shown. We should keep as much decision making out of the templates, makes it easier to maintain in the future. In the controller you could pass into the template , showLFPWarning: company.isAccountsOverdue

Reply:
Done